test/envs/kubevirt.yaml

```diff
@@ -13,10 +13,10 @@ profiles:
         io.containerd.grpc.v1.cri:
           device_ownership_from_security_context: true
     network: $network
-    cpus: 4
-    memory: "8g"
+    cpus: 8
+    memory: "16g"
     extra_disks: 1
-    disk_size: "50g"
+    disk_size: "100g"
```

Comment:
oh wow really? will the entire 100g get used?

Reply:
No, this is just virtual size. The bigger image we work with is 7g compressed qcow2/40g raw (with 13g of data). We store 13g in ceph, so maybe the 50g image is enough. I increased it so we avoid space issue.